test/python/qobj/test_qobj.py

```diff
@@ -182,9 +182,9 @@ def setUp(self):
                        'memory_slot_size': 8192,
                        'meas_return': 'avg',
                        'pulse_library': [{'name': 'pulse0',
-                                          'samples': [[0.0, 0.0],
-                                                      [0.5, 0.0],
-                                                      [0.0, 0.0]]}
+                                          'samples': [0j,
+                                                      0.5+0j,
+                                                      0j]}
```

Comment:
Similarly to https://github.com/Qiskit/qiskit-terra/pull/3383/files?file-filters%5B%5D=.py&file-filters%5B%5D=.txt&hide-deleted-files=true&owned-by%5B%5D=1ucian0#r388338265
```suggestion
                                          'samples': [0, 0.5, 0]}
```